lib/scsi_defs.h

```diff
@@ -3,12 +3,15 @@
 
 #include <cstdint>
 
+
 #include "absl/base/attributes.h"
 
 // The fields of structures in this namespace are arranged according to Big Endian format.
 namespace scsi_defs {
 
+  // size of structs in bytes
   constexpr uint32_t kInquiryCommandSize = 6;
+  constexpr uint32_t kInquiryDataSize = 96;
```

Comment:
Seems like we're moving away from defining size constants.

Reply:
Removed constants